src/utils/routes/addRoutes.js

```diff
@@ -0,0 +1,18 @@
+import dispatcher from './dispatcher';
+
+const addRoutes = router => (routes) => {
```

Comment:
since we're always passing `new Router()` here, in order to avoid importing Router from express in each route file you can import it once here and create new instance here, so the usage would be like so

```
export default addRoutes([...]):
```

and implementation is
```
const addRoutes = (routes) => {
   const router = new Router();
   routes.forEach((routeConfig) => { ... });
   return router;
}
```

One less closure again. And in case we need to extend and rouse some other route (do not create a new instance), we can resolve it with a condition later.